pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -659,8 +659,9 @@ func TestPipelineSpec_Validate_Failure(t *testing.T) {
 			}},
 		},
 		expectedError: apis.FieldError{
-			Message: `invalid value: pipeline declared resources didn't match usage in Tasks: Didn't provide required values: [missing-great-resource missing-wonderful-resource]`,
-			Paths:   []string{"resources"},
+			Message: `Resources field is deprecated in v1 Pipeline: Resources
+invalid value: pipeline declared resources didn't match usage in Tasks: Didn't provide required values: [missing-great-resource missing-wonderful-resource]`,
+			Paths: []string{"resources"},
```

Comment:
This test will need to ensure the error returned is at a warning level, otherwise the Pipeline will actually be rejected. This will probably require addressing https://github.com/knative/pkg/issues/2514